noir/noir-repo/compiler/noirc_evaluator/src/brillig/brillig_gen/brillig_directive.rs

```diff
@@ -53,16 +67,29 @@ pub(crate) fn directive_invert() -> GeneratedBrillig {
 ///    (a/b, a-a/b*b)
 /// }
 /// ```
-pub(crate) fn directive_quotient(bit_size: u32) -> GeneratedBrillig {
+pub(crate) fn directive_quotient(mut bit_size: u32) -> GeneratedBrillig {
     // `a` is (0) (i.e register index 0)
     // `b` is (1)
+    if bit_size > FieldElement::max_num_bits() {
+        bit_size = FieldElement::max_num_bits();
+    }
```

Comment:
brillig_directive was receiving 255 in bit_size for the quotient directive from acir_gen, which doesn't make sense to me. I think we might have a blind +1 in bit_size upstream